android/java/org/chromium/chrome/browser/crypto_wallet/fragments/onboarding/OnboardingFingerprintUnlockFragment.java

```diff
@@ -86,14 +88,16 @@ protected boolean canNavigateBack() {
 
     @SuppressLint("MissingPermission")
     @RequiresApi(api = Build.VERSION_CODES.P)
-    private void setUpBiometric(@NonNull final String password) {
+    private void setUpBiometric(@NonNull final String password, @NonNull final Cipher cipher) {
         final BiometricPrompt.AuthenticationCallback authenticationCallback =
                 new BiometricPrompt.AuthenticationCallback() {
                     @Override
                     public void onAuthenticationSucceeded(
                             BiometricPrompt.AuthenticationResult result) {
                         super.onAuthenticationSucceeded(result);
-                        KeystoreHelper.useBiometricOnUnlock(password);
+                        Cipher cipher = result.getCryptoObject().getCipher();
```

Comment:
why do we need to pass `@NonNull final Cipher cipher` if we use a `cipher` from `result` here?

Reply:
nvm, I see we use it later in the function. I would rename `Cipher cipher` to something `Cipher resultCipher` to avoid confusion

Reply:
Yes, I'm going to fix this also for `BaseWalletNextPageFragment`.